save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/CreateProjectProblemView.kt

```diff
@@ -51,13 +51,13 @@ val createProjectProblem: FC<CreateProjectProblemViewProps> = FC {props ->
 
     val enrollCheckVulnerabilityRequest = useDeferredRequest {
         val response = get(
```

Comment:
suggestion to use `com.saveourtool.save.frontend.utils.RequestUtilsKt#get(com.saveourtool.save.frontend.utils.ComponentWithScope<?,?>, java.lang.String, T, org.w3c.fetch.Headers, kotlin.jvm.functions.Function2<? super kotlin.jvm.functions.Function1<? super kotlin.coroutines.Continuation<? super org.w3c.fetch.Response>,? extends java.lang.Object>,? super kotlin.coroutines.Continuation<? super org.w3c.fetch.Response>,? extends java.lang.Object>, kotlin.jvm.functions.Function1<? super org.w3c.fetch.Response,kotlin.Unit>, kotlin.coroutines.Continuation<? super org.w3c.fetch.Response>)`

It allows to set parameters to `get` method instead of making string concatenation

Reply:
Fixed